### ISSUE TITLE
Document functionality of triggers property on Command resource

### ISSUE THREAD

**Issue by clstokes** [pulumi/pulumi-command#40]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

The `triggers` property on `local.Command` does not have descriptive text in the docs or in the SDK so it's not clear what is the expected behavior of this property.

![screenshot-2022-02-02-12-21-04@2x](https://user-images.githubusercontent.com/293763/152231165-82dd5da4-6dd1-45dd-bcf9-75860e301ec1.png)

**Comment by lukehoban** [comment]
This is available now:

<img width="798" alt="Screenshot 2022-12-28 at 5 21 32 PM" src="https://user-images.githubusercontent.com/223467/209891707-c9b84c1b-713b-4591-9d23-67b4a722c5df.png">

**Comment by C0DK** [comment]
This isn't that helpful. what is the type expected?

I put a FileArchive and got a `unrecognized signature '' in property map for "triggers"` back. @lukehoban

**Comment by renepetersen1984** [comment]
I used the `triggers` parameter like shown below. When the modification time for `some_file` changes, it triggers a replacement.

```
import os

copy_file = pulumi_command.remote.CopyFile(
    "copy_file",
    connection=connection,
    local_path="some_file",
    remote_path="/home/user/some_file",
    triggers=[
        os.path.getmtime("some_file")
    ]
)
```

**Comment by C0DK** [comment]
Maybe it would make sense to write something like:

`Changes to any of these values, trigger replacements` as well as the types supported :) afaik it is only simple types (maybe only strings?)

**Comment by thomas11** [comment]
I was struggling with this as well. Re-opening to improve the docs.

**Comment by HandOfGod94** [comment]
@thomas11 , @C0DK , is the `triggers` a state/flag variable, which if changes (current-run value is different for previous-runs), the underlying action gets executed?

**Comment by thomas11** [comment]
> @thomas11 , @C0DK , is the `triggers` a state/flag variable, which if changes (current-run value is different for previous-runs), the underlying action gets executed?

Yes, exactly. It's simply an opaque value of type any/object that's checked for equality with the previous value.
